### PR TITLE
Week2 Homework

### PR DESCRIPTION
**Pull request by anastasiya-kliushkina**
- Added Map Slider composable and a new star image for the plot point
- Updated theme and colors
- Updated README.md
- Added About screen and navigation to PlotSurface screen
- Added Splash screen

coordplot_with_splash_and_about

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+### Simple Coordinate Plotter app
+
+Move the Star by dragging sliders!
+
+Each slider shows percentage for the updated value on the Map.
+
+### Improvements:
+- Moved several composables into individual files for better visibility and maintenance
+- Added percentage update on each slider change
+- Updated theme and colors
```
Comment (zlippard) [inline README.md:12]: Great work utilizing the README to explain what the app is for and what improvements were added!

File: app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt [hunk 2]
```diff
@@ -3,40 +3,46 @@ package com.kodeco.android.coordplot
...
+@Composable
+fun MainScreen() {
+    //Keep track of the state of the screens and the back stacks of the screens it manages
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = "aboutscreen") {
+        composable("aboutscreen") { AboutScreen(onNextClick = { navController.navigate("plotsurfacescreen") }) }
+        composable("plotsurfacescreen") { PlotSurfaceScreen() }
+
+    }
 }
```
Comment (zlippard) [inline app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt:48]: Nice work going above and beyond here! This is definitely the right approach with multiple screens in pure compose! 👏

**Review by zlippard — APPROVED** [review]
Awesome job polishing up the app this week and styling the components, colors and typography!

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,9 @@
 <resources>
-    <string name="app_name">My Application</string>
+    <string name="app_name">Coordinate Plot</string>
+    <string name="x_axis_text">X Axis:</string>
+    <string name="y_axis_text">Y Axis:</string>
+    <string name="about_title_text">Coordinate Plot</string>
+    <string name="about_app_title">About Coordinate Plot</string>
+    <string name="about_app_text">Move the Star on the Map with sliders!</string>
+    <string name="next_button_text">Next</string>
 </resources>
```
Comment (zlippard) [inline app/src/main/res/values/strings.xml:9]: Small note on the EOF symbol at the end of some these files:  

Screen Shot 2023-09-05 at 8 54 01 PM

Typically all files need an extra newline at the end.  You can set this up in Android Studio via Preferences > Editor > General and in the On Save section ensure that the `Ensure every saved file ends with a line break` is checked.

Screen Shot 2023-09-05 at 8 54 58 PM

Comment (anastasiya-kliushkina) [reply app/src/main/res/values/strings.xml:9]: Thanks @zlippard for the review and useful tip! :)

**Pull request merged**